app.rb

```diff
@@ -126,7 +126,7 @@ def run
       print 'Select an option: '
       choice = gets.chomp.to_i
 
-      result = process_choice(choice)
+      result = process(choice)
```

Comment:
- [ ] As suggested by the previous reviewer please fix the create person feature in your console app. Currently when we try to create a student and try to add the classroom it gives an error. Please refer to the following screenshot for error's details

<img width="855" alt="Screenshot 2023-08-12 at 7 21 04 PM" src="https://github.com/Gardimy/OOP-school-library/assets/44798044/5ed4cfc9-0022-4e62-b875-a8a8b32bad54">